content/getting-started/setup/upgrades/index.md

```diff
@@ -8,4 +8,63 @@ dnnversion: 09.04.03
 # DNN Platform Upgrades
 Following are the basic steps for upgrading **DNN Platform**.
 
-Please click the **Improve this Doc** button above to help us improve this page.
\ No newline at end of file
+> [!NOTE]
+> Every upgrade scenario is unique in some shape, form or fashion. This document is by no means intended to address every scenario. Some upgrades can be quite complex and it is virtually impossible to cover every scenario. This document is intended to focus only on the upgrade basics.
+
+## Step 1 - Prepare
+Before starting the upgrade process, it is recommended to have a very clear understanding of all upgrade requirements and expectations. Here are some tips to help prepare.
+
+* Know the current version of DNN Platform.
+
+* Ensure **AutoUpgrade** setting is "False". This setting can be found within **AppSettings** section of the `web.config` file in the root. Setting this to "False" will prevent the upgrade process from triggering by normal site visitors. 
+
+* Know the desired version of DNN Platform and download the appropriate "Upgrade" ZIP package from [the official repository on GitHub](https://github.com/dnnsoftware/DNN.Platform/releases) (which can be found in the **Assets** section for the desired version release).
+
+* Know the [suggested upgrade path](xref:setup-upgrades-suggested-upgrade-path) from the current version of DNN Platform and download all appropriate "Upgrade" ZIP packages from the above referenced GitHub link.
+
+* Know any changes to the minimum requirements for the desired version of DNN Platform and plan for implementation accordingly. 
+
+* Know upgrade requirements (if applicable) for any installed third-party extensions and plan for implementation accordingly.
+
+* If possible, plan to test the upgrade in a non-production environment prior to attempting in production.
+
+## Step 2 - Take Website Offline
+When ready to begin the upgrade process, take the website offline to avoid any access or changes to the site by other users. This can be done by adding an `app_offline.htm` file to the root of the site. This is an ASP.NET feature and the HTML file can contain any HTML you see fit to communicate with site visitors about the site unavailability. 
+
+> [!NOTE]
+> This also prevents access to any other site pages. Any attempts will be redirected to the base URL and will display the HTML message contained in the `app_offline.htm` file.
+
+## Step 3 - Backup Files & Database
+It is recommended to perform this step in a "native" fashion.
```

Comment:
It is unclear what `native fashion` means.

Reply:
See the sentences following that one.  That said, if you have a recommendation for better wording, feel free to change.

Reply:
After re-reading it fully, I agree - it is a clunky choice of words.  I will rewrite and submit in a new PR.